ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -218,23 +212,24 @@ export function AlertModal({
     <Modal isOpen onClose={handleClose}>
       <ModalOverlay />
       <ModalContent>
-        <ModalHeader onClose={handleClose}>
-          <Header selectedAlert={selectedAlert} />
-        </ModalHeader>
+        <ModalHeader
+          onClose={handleClose}
+          startAccessory={headerStartAccessory}
+          className={'alert-modal__header'}
+          borderWidth={1}
+          display={headerStartAccessory ? Display.InlineFlex : Display.Block}
+        />
         <ModalBody>
+          <AlertHeader selectedAlert={selectedAlert} />
```

Comment:
Is this significant, why have we moved our header inside the body?

Reply:
Moved on top of the `<ModalBody>`, with no difference in the design.